### DISCUSSION TITLE
Disable splitting of tomosynthesis data in explorer

### DISCUSSION THREAD

**Discussion by res80** [nroduit/Weasis#508] [Q&A]
Hello!
When I load images from our tomosynthesis, each slice is shown as a separate sub-series in the dicom explorer.
I know that I can select all sub-series and merge them to have a single series again.
However, this is not so easy for unexperienced users.
Is there a way to change this behaviour for tomosynthesis data?
Change a script within weasis?
Kind regards,
Ruediger

**Comment by nroduit** [comment]
First you need to determine which tag causes the separation in the configuration file according to the modality (and whether it's a mulitframe). To test this, the [series-splitting-rules.xml](https://github.com/nroduit/Weasis/blob/master/weasis-distributions/resources/series-splitting-rules.xml) file can be modified (uncommenting the rules) in the "resources" directory where the installation is located.

See also this [issue](https://github.com/nroduit/Weasis/issues/442)
